### DISCUSSION TITLE
Hide/show Categorys and Menu-Buttons for groups (Roles)

### DISCUSSION THREAD

**Discussion by ghost** [gameserverapp/Platform#250] [Suggestions]
Would be nice if forum category and menu buttons could be set to only be visible to certain groups (roles)
(e.g.: admin, members, donors, etc.). So you could set it up so that only "members" see the button for the shop.
Or that only the admins can see a button for the dashboard.
Unfortunately, my English is not that good. I hope you can understand my thoughts here.

**Comment by mvaessen** [comment]
Note for later: This requires the Community API to include roles. From there people can use the Open source community website to add this functionality as they like

**Comment by ImpulsiveFPS** [comment]
Would love this. It would also be nice with a option to only show certain pages for logged in users.

**Comment by zerafox** [comment]
I would love to have private categories/threads for just the Admins or certain roles.
Right now setting something to "private" I'm not even sure what that does..
